apps/portal/src/app/insight/layout.tsx

```diff
@@ -0,0 +1,16 @@
+import { DocLayout } from "@/components/Layouts/DocLayout";
+import { createMetadata } from "@doc";
+import { sidebar } from "./sidebar";
+
+export default async function Layout(props: { children: React.ReactNode }) {
+  return (
+    <DocLayout sideBar={sidebar} editPageButton={true}>
+      {props.children}
+    </DocLayout>
+  );
+}
+
+export const metadata = createMetadata({
+  title: "Insight",
+  description: "Query, transform and analyze blockchain data",
+});
```

Comment:
Please also set `image` key for adding an OG image

Example: 
```
image: {
 title: "Insight",
 icon: "thirdweb"
}
```

You can also add a new icon for "insight" in `createMetadata` function

Reply:
Added icon for insight and updated the image prop in metadata